test/Compressors/Gaussian.jl

```diff
@@ -0,0 +1,157 @@
+module Gaussian_compressor
+    using Test, RLinearAlgebra
+    import Base.:*
+    import LinearAlgebra: transpose, adjoint
+    import LinearAlgebra: mul!, lmul!
+    import Random: randn!, seed!
+    using ..FieldTest
+    using ..ApproxTol
+
+    @testset "Compressor_Gaussian" begin
+        @test_compressor GaussianRecipe
+        let 
+            seed!(21321)
+            S1 = Gaussian()
+            @test typeof(S1) <: Compressor
+            @test S1.cardinality == Left
+            @test S1.compression_dim == 2
+            # Test the case when the compression_dim input is not positive
+            @test_throws ArgumentError Gaussian(cardinality = Left, compression_dim = -7)
+        end
```

Comment:
May want to separate these out into two blocks as they are testing for different things. The first one is testing for default behavior of the constructor while the second is constructing a new Compressor to see if it throws an error. Should be testing the edge case (i.e., `compression_dim=0`) as well